.cirrus.yml

```diff
@@ -116,6 +116,12 @@ gating_task:
         - '/usr/local/bin/entrypoint.sh podman-remote-darwin |& ${TIMESTAMP}'
         - '/usr/local/bin/entrypoint.sh podman-remote-windows |& ${TIMESTAMP}'
 
+    # Verify some aspects of ci/related scripts
+    ci_script:
+	    - '${CIRRUS_WORKING_DIR}/${SCRIPT_BASE}/lib.sh.t |& ${TIMESTAMP}'
```

Comment:
GH is complaining about indentation here?

Reply:
Ohh! There's a tab in there, yeah that will mess it up badly.  Thanks.